src/test/mochitest/browser_dbg-expressions-error.js

```diff
@@ -21,18 +21,14 @@ function getValue(dbg, index) {
   return findElement(dbg, "expressionValue", index).innerText;
 }
 
-function assertEmptyValue(dbg, index) {
-  const value = findElement(dbg, "expressionValue", index);
-  if (value) {
-    is(value.innerText, "");
-    return;
-  }
-
-  is(value, null);
-}
-
 function toggleExpression(dbg, index) {
-  findElement(dbg, "expressionNode", index).click();
+  const node = findElement(dbg, "expressionNode", index);
+  const objectInspector = node.closest(".object-inspector");
+  const properties = objectInspector.querySelectorAll(".node").length;
+  node.click();
+  return waitUntil(
+    () => objectInspector.querySelectorAll(".node").length !== properties
+  );
 }
```

Comment:
Perhaps we can have helpers like this in `mochitest/head.js`?

```js
async function toggleExpression() {
  const objectInspector = findElement(dbg, "expressionNode", index).closest(".object-inspector");
  await toggleObjectInspectorNode(dbg, objectInspector);
}

async function toggleObjectInspectorNode(dbg, objectInspector) {
   const getNumProperties = () => objectInspector.querySelectorAll(".node").length;
   const numProperties = getNumProperties();
   node.click();
   return waitUntil(() => numProperties !== getNumProperties());
}
```

Reply:
yeah, sounds like a good idea

Reply:
✅